hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/QueuePath.java

```diff
@@ -59,6 +60,15 @@ public QueuePath(String fullPath) {
     setFromFullPath(fullPath);
   }
 
+  /**
+   * Concatenate queue path parts into one queue path string.
+   * @param parts Parts of the full queue pathAutoCreatedQueueTemplate
+   * @return full path of the given queue parts
+   */
+  public static String concatenatePath(String... parts) {
```

Comment:
This method is unused. Please remove it.

Reply:
This is used at here: TestRMWebServicesCapacitySched@verifySubQueueXML at line 339

for (int k = 0; k < qListInfos.getLength(); k++) {
  Element qElem3 = (Element) qListInfos.item(k);
  String qName3 = WebServicesTestUtils.getXmlString(qElem3, "queueName");
  **String q3 = QueuePath.concatenatePath(queue, qName3);**
  verifySubQueueXML(qElem3, q3, qi.absoluteCapacity, qi.absoluteMaxCapacity);
}

Reply:
I think that code fragment has been removed.

Reply:
Yep, there is no caller of this. Please make sure to add this to the follow-up jira's description as well.